risiko.cpp

```diff
@@ -0,0 +1,96 @@
+#include <iostream>
+#include <cstdlib>
+#include <string>
+using namespace std;
+
+void switchSort(int* , int);
+
+class Player{
+    private:
+        string role;
+
+    public:
+
+        enum Color{
+            RED, BLUE
+        };
+
+        Color side;
+        Player(Color c, string str);
+
+        string getRole()const{ return role;}
+        int makeLaunch()const{ return rand()%(6-1+1)+1;}
+
+};
+
+
+Player::Player(Color c, string str){
+    this->side = c;
+    this->role = str;
+}
+
+
+void results(int* arr){
+    cout << arr[2] <<"(N)\n";
+    cout << arr[1] <<"(M)\n";
+    cout << arr[0] <<"(O)\n";
+}
+
+
+static void match(Player& x, Player& y, int* arr1, int* arr2){
+    switchSort(arr1, 3);
+    switchSort(arr2, 3);
+
+    cout << "Red dices:" << endl;
+    results(arr1);
+    cout << endl;
+    cout << "Blue dices" << endl;
+    results(arr2);
+
+    cout << endl << endl;
+    string s = "OMN";
+
+    for(int i = 2; i >= 0; i--){
+        if(arr1[i] <= arr2[i]){
+            cout << s[i] <<" "<< arr1[i] <<"\tvs\t" << arr2[i] << "   " << "==>   ";
+            cout << "blue wins\n";
```

Comment:
per `consistency` usa `\n` oppure solo `endl`, in modo da mantenere coerenza all'interno del codice
se uno sviluppatore leggesse questo codice non saprebbe cosa usare tra i due